ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/azuremag/cucushift-installer-rehearse-azure-ipi-azuremag-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: cucushift-installer-rehearse-azure-ipi-azuremag
+  steps: 
+    pre:
+    - chain: ipi-conf-azuremag
+    - chain: ipi-install
+    post:
+    - chain: gather-network
```

Comment:
Lets add `-chain: gather` here to include must-gather, extra and audit log steps?

Reply:
When I add `- chain: gather` I get this error when I run `make jobs`
`
level=fatal msg="failed to parse arguments" error="failed to load registry: [workflow/cucushift-installer-rehearse-azure-ipi-azuremag: chain/ipi-deprovision: chain/gather: duplicate name: gather-must-gather, workflow/cucushift-installer-rehearse-azure-ipi-azuremag: chain/ipi-deprovision: chain/gather: duplicate name: gather-extra, workflow/cucushift-installer-rehearse-azure-ipi-azuremag: chain/ipi-deprovision: chain/gather: duplicate name: gather-audit-logs]"
`
What should I do?

Reply:
When I add `- chain: gather` I get this error when I run `make jobs`
`
level=fatal msg="failed to parse arguments" error="failed to load registry: [workflow/cucushift-installer-rehearse-azure-ipi-azuremag: chain/ipi-deprovision: chain/gather: duplicate name: gather-must-gather, workflow/cucushift-installer-rehearse-azure-ipi-azuremag: chain/ipi-deprovision: chain/gather: duplicate name: gather-extra, workflow/cucushift-installer-rehearse-azure-ipi-azuremag: chain/ipi-deprovision: chain/gather: duplicate name: gather-audit-logs]"
`
What should I do?

Reply:
Ah, its already a part of `ipi-deprovision` chain, good